docs/modules/provisioning-app/pages/configuration.adoc

```diff
@@ -413,10 +413,10 @@ To disable the confluence adapter you can add this property to the application p
 adapters.confluence.enabled=false
 ```
 
-The Openshift Service can be used to verify that a project key does not exist in the cluster before provisioning a project.
-If you want to enable it, you can add this property to the application properties:
+The Openshift Service currently is used to verify that a project key does not exist in the cluster before provisioning a project.
+If you need to disable it, you can add this property to the application properties:
 ```
-services.openshift.enabled=true
+services.openshift.enabled=false
```

Comment:
I would keep this feature disabled until we have the chance to test it in openshift, or did you have the chance to deploy this version of prov app in openshift?

Reply:
excellent! Let us merge this change and the ODS build run the E2E tests. Then we can verify that nothing breaks before porting this change to branch `4.x`. Sounds good?